usb/usb-skeleton.c

```diff
@@ -119,7 +119,7 @@ static ssize_t skel_read(struct file *file, char __user *buffer, size_t count, l
 			      usb_rcvbulkpipe(dev->udev, dev->bulk_in_endpointAddr),
 			      dev->bulk_in_buffer,
 			      min(dev->bulk_in_size, count),
-			      &count, HZ*10);
+			      (int *) count, HZ*10);
```

Comment:
You shouldn't remove the `&`, otherwise you are passing the count variable by value while what you want is to pass it by reference.